index.html

```diff
@@ -595,6 +595,32 @@ <h3 id="profile-description-methodology">Methodology</h3>
     </section>
   </section>
 
+  <section id="device-categoriesdevices">
+    <h2>Device Categories</h2>	  
+<p>
+  In a deployment of WoT conforming to the <a href="https://www.w3.org/TR/wot-architecture/#architecture-abstract">WoT Abstract Architecture</a>
+  we see different device types with typical characteristics.
+  They range (sorted in the order of footprint and capabilities) 
+  from small embedded <em>node</em> devices to <em>gateways</em> or <em>hubs</em> 
+  to powerful <em>edge</em> devices and <em>cloud</em> servers.
+  Interoperability between these devices implies that a core set of features and functionalities
+  is available on <em>all</em> of them.
+</p>
+
+<p>The following device categories describe the footprint and characteristics of typical representants of these classes.
+ This is used to identify the possible features and use cases for these device classes.
+
+ <ul>
+ <li>Node - endpoint with typically 64-128 KB of RAM</li>
```

Comment:
**node** only is unspecific. **resource constrained** node 

>typically 64-128 KB of RAM

 I'm not sure about such a range. I would simple say typically KB of RAM